### PR TITLE
Cleanup deprecated style

### PR DESCRIPTION
**Pull request by hsbt**
# Description:

Followed up to remove old ruby versions. I did clean up redundant/deprecated sections on gemspec and travis.yml.
______________

# Tasks:

- [ ] Describe the problem / feature
- [ ] Write tests
- [ ] Write code to solve the problem
- [ ] Get code review from coworkers / friends

I will abide by the [code of conduct](https://github.com/rubygems/rubygems/blob/master/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Comment by hsbt** [comment]
@bundlerbot r+

**Comment by hsbt** [comment]
@bundlerbot r+

**Comment by hsbt** [comment]
@bundlerbot retry

File: rubygems-update.gemspec [hunk 1]
```diff
@@ -1,44 +1,36 @@
-# -*- encoding: utf-8 -*-
-# stub: rubygems-update 2.7.4 ruby hide_lib_for_update
-
 Gem::Specification.new do |s|
   s.name = "rubygems-update".freeze
   s.version = "2.7.4"
```
Comment (segiddins) [inline rubygems-update.gemspec]: should this be reading from `lib/rubygems.rb`?

File: rubygems-update.gemspec [hunk 2]
```diff
@@ -1,44 +1,36 @@
...
+  s.summary = "".freeze
+  s.description = "".freeze
   s.homepage = "https://rubygems.org".freeze
   s.licenses = ["Ruby".freeze, "MIT".freeze]
+
+  s.files = File.read('Manifest.txt').split
+  s.executables = ["update_rubygems".freeze]
+  s.require_paths = ["hide_lib_for_update".freeze]
   s.rdoc_options = ["--main".freeze, "README.md".freeze, "--title=RubyGems Update Documentation".freeze]
+  s.extra_rdoc_files = [
```
Comment (segiddins) [inline rubygems-update.gemspec]: should this include the bundler manpages?

File: rubygems-update.gemspec [hunk 3]
```diff
@@ -1,44 +1,36 @@
...
+    "History.txt".freeze, "LICENSE.txt".freeze, "MAINTAINERS.txt".freeze,
+    "MIT.txt".freeze, "Manifest.txt".freeze, "README.md".freeze,
+    "UPGRADING.rdoc".freeze, "POLICIES.rdoc".freeze, "CODE_OF_CONDUCT.md".freeze,
+    "CONTRIBUTING.rdoc".freeze, "bundler/CHANGELOG.md".freeze, "bundler/CODE_OF_CONDUCT.md".freeze,
+    "bundler/CONTRIBUTING.md".freeze, "bundler/LICENSE.md".freeze, "bundler/README.md".freeze,
+    "hide_lib_for_update/note.txt".freeze,
+  ]
+
   s.required_ruby_version = Gem::Requirement.new(">= 2.2.2".freeze)
   s.rubygems_version = "2.7.3".freeze
```
Comment (segiddins) [inline rubygems-update.gemspec]: I believe this should be removed, it's set by RubyGems when building

**Pull request merged**